aten/src/ATen/native/cuda/int4mm.cu

```diff
@@ -125,7 +125,11 @@ inline __host__ __device__ uint32_t getAlignmentRoundUp(const void* p) {
   return diff == 0 ? 0 : uint32_t(Align) - diff;
 }
 
+#if defined(USE_ROCM)
+constexpr int32_t kWarpSize = __AMDGCN_WAVEFRONT_SIZE;
```

Comment:
`warpSize` is a constexpr for HIP.  Prefer over internal `__AMDGCN_WAVEFRONT_SIZE`.

Reply:
thanks. Can you share how does that work?
quick grep but couldn't find that in ROCm/HIP...

Reply:
https://github.com/ROCm/clr/blob/7ad141646420fb275d8d7358521f7806a92cda9a/hipamd/include/hip/amd_detail/amd_warp_functions.h#L76

__AMDGCN_WAVEFRONT_SIZE is defined by hip-clang compiler.  For device passes it will be set correctly based on gfx target.  For host passes, it defaults to 64.

Reply:
https://clang.llvm.org/docs/HIPSupport.html#predefined-macros

Reply:
Thanks, so it's from clr. `static constexpr int warpSize = __AMDGCN_WAVEFRONT_SIZE;`

Let me switch to `warpSize` if thats preferred and fix the comments as well